engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/immutable/FromPolyglotArrayBuiltinVectorNode.java

```diff
@@ -4,15 +4,17 @@
 import com.oracle.truffle.api.interop.InteropLibrary;
 import com.oracle.truffle.api.library.CachedLibrary;
 import com.oracle.truffle.api.nodes.Node;
-import org.enso.interpreter.dsl.*;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.dsl.Owner;
 import org.enso.interpreter.runtime.Context;
 import org.enso.interpreter.runtime.error.PanicException;
 import org.enso.interpreter.runtime.data.Vector;
 
 @BuiltinMethod(
     type = "Vector",
     name = "from_polyglot_array",
-    description = "Returns an Array representation of this Vector.")
+    description = "Returns an Array representation of this Vector.",
```

Comment:
It's not your change, but this description seems wrong. Can we fix it?

```suggestion
    description = "Creates a Vector by providing its underlying storage as a polyglot array. The underlying array should be guaranteed to never be mutated.",
```